sygnal/sygnal.py

```diff
@@ -76,6 +76,17 @@ def __init__(self, config, custom_reactor, tracer=opentracing.tracer):
         observer = twisted_log.PythonLoggingObserver()
         observer.start()
 
+        # Old format db config
+        if config.get("db", None) is not None:
```

Comment:
`dict.get` defaults to None, and it's pythonic to skip the explicit `is not None`

```suggestion
        if config.get("db"):
```

Reply:
Good to know :+1: